addons/FP_UniformsEverywhereElse/config.cpp

```diff
@@ -50,7 +50,8 @@ class CfgPatches
 			"A3_Characters_F_Beta",
 			"A3_Characters_F",
 			"A3_Characters_F_BLUFOR",
-			"A3_Data_F_Oldman_Loadorder"
+			"A3_Data_F_Oldman_Loadorder",
+			"jam_characters_east"
```

Comment:
fyi this will add jamie's uniforms as requirement. I would suggest putting that into its own pbo/config in case we remove jamies to cut down on maintenance cost.